integration/tests/accessibility.spec.ts

```diff
@@ -65,6 +66,7 @@ test.describe('axe should not find any automatically detectable accessibility is
 
 			const accessibilityScanResults = await new AxeBuilder({ page })
 				.exclude('#figma-embed-iframe')
+				.exclude('#embed-iframe')
```

Comment:
How necessary is the `iframe` example? Given it isn't accessible by default, and that we can show outcomes with `video` or `img`, could we just mention it works with iframes instead of adding one?

Reply:
Good point, removed